yt/visualization/plot_modifications.py

```diff
@@ -54,11 +55,11 @@ def _check_geometry(self, plot):
 
 def _validate_factor_tuple(factor) -> Tuple[int, int]:
     if (
-        is_sequence(factor)
+        isinstance(factor, Sequence)
         and len(factor) == 2
         and all(isinstance(_, Integral) for _ in factor)
```

Comment:
for clarity, this is an actual bug fix: previously we were checking that `factor` supported `__len__`, but not `__getitem__`, as a result, `all(isinstance(_, Integral) for _ in factor)` could fail to evaluate, raising an impromptu `TypeError` in the middle of a sanitising function